src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -36,22 +44,21 @@ export const AdminVenueView: React.FC = () => {
   const { isCurrentVenueLoaded } = useConnectCurrentVenueNG(venueId);
 
   const renderAdminVenueTabs = useMemo(() => {
-    return Object.entries(adminVenueTabLabelMap).map(([key, text]) => (
-      <Nav.Link
-        key={key}
-        className={classNames("AdminVenueView__tab", {
-          "AdminVenueView__tab--selected": selectedTab === key,
-        })}
-        eventKey={key}
-      >
-        {text}
-      </Nav.Link>
+    return Object.entries(adminVenueTabLabelMap).map(([key, label]) => (
+      <LinkContainer key={key} to={adminNGVenueUrl(venueId, key)}>
+        <Nav.Link
+          key={key}
+          className={classNames("AdminVenueView__tab", {
+            "AdminVenueView__tab--selected": selectedTab === key,
+          })}
+          eventKey={key}
+          href={adminNGVenueUrl(venueId, key)}
+        >
+          {label}
+        </Nav.Link>
+      </LinkContainer>
```

Comment:
NB:  `LinkContainer` irons out the inconsistencies between Bootstrap's `Nav.Link` and React Router's `Link` and provides the same interface/props as the React Router one.

Reply:
If you replace `Nav.Link` with `Link` There would be no boostrap in this chunk of code, hence to need for the library, am I missing something else? What are these inconsistencies ?

Reply:
My bad, misread that `Nav.Link`.

Reply:
Removed dependency and `Nav.Link`s at ba2dd1d5395275e3a9cc516552882a888407661e as per https://doesitsparkle.slack.com/archives/C01H9D1P1PE/p1626860925271700?thread_ts=1626860598.271300&cid=C01H9D1P1PE